code/game/machinery/computer/Operating.dm

```diff
@@ -52,13 +52,13 @@
 	for(var/direction in GLOB.alldirs)
 		var/obj/structure/table/optable/found_table = locate(/obj/structure/table/optable) in get_step(src, direction)
 		if(found_table && (!found_table.computer || found_table.computer == src))
-			return table
```

Comment:
such an easy-to-miss yet devastating (well, as in, it broke the whole functionality of this proc) typo, lol